sentry_sdk/integrations/fastapi.py

```diff
@@ -0,0 +1,122 @@
+from sentry_sdk._types import MYPY
+from sentry_sdk.hub import Hub
+from sentry_sdk.integrations import DidNotEnable
+from sentry_sdk.integrations.starlette import (
+    SentryStarletteMiddleware,
+    StarletteIntegration,
+)
+from sentry_sdk.tracing import SOURCE_FOR_STYLE, TRANSACTION_SOURCE_ROUTE
+from sentry_sdk.utils import transaction_from_function
+
+if MYPY:
+    from typing import Any, Callable, Dict
+
+    from sentry_sdk._types import Event
+
+try:
+    from fastapi.applications import FastAPI
+    from fastapi.requests import Request
+except ImportError:
+    raise DidNotEnable("FastAPI is not installed")
+
+try:
+    from starlette.types import ASGIApp, Receive, Scope, Send
+except ImportError:
+    raise DidNotEnable("Starlette is not installed")
+
+
+_DEFAULT_TRANSACTION_NAME = "generic FastApi request"
```

Comment:
Just a tiny nitpick, but it would be cool if this used the spelling `FastAPI`, with the "API" part in upper case.

Reply:
done

Reply:
(in some other branch)

Reply:
Thanks!